src/ADAL.PCL.Android/CryptographyHelper.cs

```diff
@@ -34,6 +34,11 @@ internal class CryptographyHelper : ICryptographyHelper
     {
         public string CreateSha256Hash(string input)
         {
+            if (string.IsNullOrWhiteSpace(input))
+            {
+                return null;
+            }
+
             return string.Empty;    // TODO: Update
```

Comment:
This check exists the way it is because, we do not expect developers to use confidential client on native applications for obvious security reasons. this exception (not thrown by intent) at least prevented the developer to do wrong things :)